docs/concepts/resources/filter_types.py

```diff
@@ -219,4 +219,23 @@
 axis.set_ylim(-70, 20)
 axis.legend(loc=4, ncol=2)
 
+# notch filter
```

Comment:
maybe plot the 3 figures in same style as the Filter banks.

Reply:
You mean 3 by 1 subplot layout?